sandbox/concurrent_bert_etl/app_all_lambdas/jobs.py

```diff
@@ -0,0 +1,232 @@
+"""This module defines all the Lambda functions to be deployed to AWS.
+The function name would be AWS Lambda function name as well. Everything to be
+packaged to Lambda must be contained *inside* the function. Module level
+imports are for local testing or packaging use only.
+
+Use ``bert.constants.AWS_LAMBDA_FUNCTION`` to check if function is running
+locally or in AWS.
+
+"""
+from bert import binding, constants, utils
+
+
+@binding.follow('noop')
+def bert_tess_fullframe_main():
+    """Extract light curve data from TESS full frame images for a given
+    TESS observation ID.
+
+    This Lambda function loops through full frame images. For each full frame
+    image, it invokes :func:`bert_tess_fullframe_worker`.
+
+    """
+    import time
+    from astroquery.mast import Observations
+
+    # TODO: Grab this from Lambda event or work_queue from a future higher
+    #       level Lambda. Example event:
+    # {
+    #   "id": "tess-s0001-1-1",
+    #   "xpos": 1170,
+    #   "ypos": 1052,
+    #   "radius": 8,
+    #   "bright_pixel_threshold": 2.5
+    # }
+    #
+    # TODO: Calculate pos, radius, and threshold from the 10th frame?
+    event = {"id": "tess-s0001-1-1",
+             "xpos": 1170,
+             "ypos": 1052,
+             "radius": 8,
+             "bright_pixel_threshold": 2.5}
+
+    work_queue, done_queue, ologger = utils.comm_binders(
+        bert_tess_fullframe_main)
+
+    # Find full frame dataset for the observation ID.
+    ologger.info('Started quering Observations...')
+    obs_table = Observations.query_criteria(obs_id=event['id'])
+    products = Observations.get_product_list(obs_table)
+    filtered = Observations.filter_products(
+        products, productSubGroupDescription="FFIC", mrp_only=False)
+
+    # Use AWS S3 bucket to pull data from.
+    Observations.enable_cloud_dataset(verbose=False)
```

Comment:
Note to self: Do not download/upload from/to S3 when testing locally. Add logic for that.